ArduPlane/Plane.h

```diff
@@ -735,6 +735,9 @@ class Plane : public AP_Vehicle {
         // are we trying to follow terrain?
         bool terrain_following;
 
+        // are we waiting to load terrain data to init terrain following
+        bool terrain_following_pending;
```

Comment:
@tridge: Is there any issue with this getting out of sync with `terrain_following`? Might be better to combine them into an `enum`?

Reply:
i'd be happy to change that in a future PR, yes